docs/guides/integration-prebid-client-side.md

```diff
@@ -0,0 +1,218 @@
+
+## Integration Example
+
+An example of the UID2 Prebid.js client-side integration is available at the following links:
+
+- Code: [Example Prebid.js UID2 Integration](https://github.com/IABTechLab/uid2docs/tree/main/static/examples/cstg-prebid-example)
+- Running site: [UID2 Prebid.js Client-Side Integration Example](https://unifiedid.com/examples/cstg-prebid-example/)
+
+## Integration Overview: High-Level Steps
+
+You'll need to complete the following steps:
+
+1. [Complete UID2 account setup](#complete-uid2-account-setup).
+2. [Add Prebid.js to your site](#add-prebidjs-to-your-site).
+3. [Configure the UID2 module](#configure-the-uid2-module).
+
+
+### Complete UID2 Account Setup
+
+Complete the UID2 account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page. As part of the account setup process for a client-side implementation, you'll need to provide a list of domain names for the sites that you'll be using with Prebid.js.
+
+:::tip
+Only root-level domains are required for account setup. For example, if you're going to use UID2 with Prebid.js on example.com, shop.example.com, and example.org, you only need to provide the domain names example.com and example.org.
+:::
+
+When account setup is complete, you'll receive a public key and subscription ID. These values are unique to you, and you'll use them to configure the UID2 module. For details, see [Subscription ID and Public Key](../getting-started/gs-credentials.md#subscription-id-and-public-key).
+
+### Add Prebid.js to Your Site
+<!-- GWH "Add Prebid.js to Your Site" section is identical for client side and server side. -->
+To add Prebid.js to your site, follow the instructions in [Getting Started for Developers](https://docs.prebid.org/dev-docs/getting-started.html) in the Prebid.js documentation. 
+
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named **Unified ID 2.0**, listed under the section **User ID Modules**.
+
+When you've added Prebid.js to your site and confirmed that it's working properly, you're ready to configure the UID2 module.
+
+:::tip
+To make sure that the UID2 module is installed, find the string `uid2IdSystem` in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
+
+### Configure the UID2 Module
+
+To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** that you received during account setup, as well as the user's hashed or unhashed email address or phone number.
+
+Once it's configured, the UID2 module generates a UID2 token for the user and stores it in the user's browser. The module automatically refreshes the token as needed as long as your site is open in the user's browser.
+
+You can configure the UID2 module to send any one of the four accepted DII formats, for any specific user:
```

Comment:
> You can configure the UID2 module to **send** any one of the four accepted DII formats
This sounds confusing - I think it should just be "You can configure the UID2 module using any one ..."

Reply:
Fixed. Same wording was in the JS Express doc -- fixed it there also.